src/accelerate/accelerator.py

```diff
@@ -560,6 +562,36 @@ def save(self, obj, f):
         """
         save(obj, f)
 
+    def save_state(self, output_dir: str):
+        """
+        Saves the current states of the model, optimizer, scaler, and RNG generators to `folder_name`.
```

Comment:
```suggestion
        Saves the current states of the model, optimizer, scaler, and RNG generators.
```
It's actually output_dir but since we document it just after, no need to mention it up there I believe.